doc/tutorials/duckdb-native-sqlalchemy.md

```diff
@@ -0,0 +1,134 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.7
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: JupySQL and DuckDB with SQLAlchemy vs native connection
+    keywords: jupyter, jupysql, duckdb, sqlalchemy
+    property=og:locale: en_US
+---
+
+# DuckDB (native vs SQLAlchemy)
+
+> tl;dr; If using DuckDB, use a native connection. Only use SQLAlchemy for legacy projects
+
+Historically, `ipython-sql` has only supported databases via SQLAlchemy. Using SQLAlchemy introduces a big overhead, when converting results to data frames. We attempted to fix this by allowing users to open a connection with SQLAlchemy while still leveraging DuckDB's highly performant capabilities to convert results into data frames; however, we encounter many edge cases, and ultimately decided to depreacate this behavior.
+
+In consequence, DuckDB connections made via SQLAlchemy suffer from the performance problem, but native connections do not. So we're now recommending users to connect to DuckDB via a native connection, this is possible since JupySQL introduced support for generic [DBAPI 2.0](https://peps.python.org/pep-0249/) drivers in version 0.7.2.
+
++++
+
+## Performance comparison (pandas)
+
+### DuckDB + SQLALchemy
+
+```{code-cell} ipython3
+import pandas as pd
+
+df = pd.DataFrame({"x": range(1_000_000)})
```

Comment:
Two suggestions here:
1. Can we add another query here (maybe add another column `y` and select `x+y`)
2. Can show a comparison table at the end for quick reference